docs/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import CountUp from 'react-countup';
+import Typewriter from 'typewriter-effect';
+import { Github } from '@brightlayer-ui/icons-mui';
+import Mail from '@mui/icons-material/Mail';
+import { SharedAppBar } from '../layout';
+import { Theme } from '@mui/system';
+import { PADDING } from '../shared/constants';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { listOfAllBluiReactPackages } from '../__configuration__/listOfAllBluiReactPackages';
+import CirclesImage from '../assets/circles.svg';
+
+const CONTENT_WIDTH = 800;
+
+const styles = {
+    pageRoot: {
+        background: (theme: Theme): string =>
+            `url('${CirclesImage}') no-repeat center/200%, ${theme.palette.primary.main}`,
+        height: '100%',
+        color: 'primary.contrastText',
+    },
+    content: {
+        maxWidth: CONTENT_WIDTH,
+        margin: (theme: Theme): string => `${theme.spacing(9)} auto`,
+        padding: `${PADDING}px`,
+    },
+    pageTitle: {
+        '& .MuiTypography-root': {
+            fontFamily: `"Roboto Mono", monospace !important`,
+        },
+        height: 200,
+    },
+    packageStats: {
+        maxWidth: (CONTENT_WIDTH - PADDING * 2) / 2,
+    },
+    buttons: {
+        contained: { color: 'primary.main', backgroundColor: 'primary.contrastText' },
+        outlined: {
+            color: 'primary.contrastText',
+            borderColor: 'primary.contrastText',
+            '&:not(.MuiTab-root):visited': {
+                color: 'primary.contrastText',
+            },
+        },
+    },
+};
+
+export const HomePage = (): JSX.Element => {
+    const lgUp = useMediaQuery((theme: Theme) => theme.breakpoints.up('lg'));
+    const [totalDownloadCount, setTotalDownloadCount] = React.useState(0);
+
+    React.useEffect(() => {
+        setTotalDownloadCount(0);
+        listOfAllBluiReactPackages.forEach((packageName) => {
+            void fetch(`https://api.npmjs.org/downloads/point/last-month/${packageName}`)
+                .then((res) => res.json())
+                .then((res) => setTotalDownloadCount((count) => count + (res.downloads as number)));
+        });
+    }, []);
+
+    return (
+        <Box sx={styles.pageRoot}>
+            <SharedAppBar title={''} />
+            <Stack sx={styles.content} gap={8}>
+                <Box sx={styles.pageTitle}>
+                    <Typography variant={lgUp ? 'h2' : 'h3'}>
+                        {lgUp && (
+                            <span
```

Comment:
Instead of using span directly, you can use MUI's Box component and pass component prop as "span" and use "sx" instead of style:
 ```
<Box
      component={'span'}
      sx={{ position: 'relative', right: '1em', top: 'calc( 1em + 10px )', opacity: 0.5 }}
  >
      $
  </Box>
```

Reply:
Thanks. I don't see any direct value in using Box and forcing it to be a span instead of just using a span, but I can go either way.